content/tools/terragrunt.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "terragrunt"
+description: "Terragrunt is a thin wrapper for Terraform that provides extra tools for working with multiple Terraform modules."
+tags:
+- terraform
+- task-runner
+- wrapper
+---
+
+Terragrunt is a well-built tool that offers a lot of functionality making terraform easier. Like all well-built tools, it's opinionated and consistent. Perhaps, one of the best things about `terragrunt` is the community around it. The community (and gruntworks) provides the documentation, best practices, and the framework for terraform operations that it pioneered. Because of terragrunt, people have taken terraform beyond its limits and taught us a lot. If a company can rely solely on the capabilities of terraform, then terragrunt is aptly suited for the task at hand.
+
+One way to think about `terragrunt` is as a purpose-built task runner (e.g. `make`) for terraform.  It supports standard capabilties of task-runners like hooks, dynamic functions, dependencies, dynamic variables and more. These are all essential for operating terraform at scale.
+
+## Problem
```

Comment:
This should be about what problem `terragrunt` solves, before we introduce our problem with terragrunt. =)